cpp/src/groupby/sort/group_std.cu

```diff
@@ -20,12 +20,14 @@
 #include <cudf/column/column_factories.hpp>
 #include <cudf/column/column_view.hpp>
 #include <cudf/detail/aggregation/aggregation.hpp>
+#include <cudf/detail/null_mask.hpp>
```

Comment:
```suggestion
```
Probably not needed?

Reply:
Yup this was for `count_nulls` before I switched to the `atomicAdd` approach.